Header header
uint16 u
uint16 v
geometry_msgs/Point pt
